### ISSUE TITLE
ox_lib not marked as a dependency

### ISSUE THREAD

**Issue by Mathu-lmn** [ND-Framework/ND_Core#71]
ox_lib is not marked as a dependency and it can lead to some issues if ox_lib is started AFTER this resource !

Issue reference on the cfx discord : https://discord.com/channels/192358910387159041/1164139686378356816/1243865784456052758

**Comment by davefl67** [comment]
Good idea to correct it in the manifest. Could head off some of the more basic questions and errors in the support Discord.

**Comment by Andyyy7666** [comment]
Thank you for creating this issue
